examples/nidcpower/measure-record.py

```diff
@@ -103,23 +88,20 @@ def check_for_initialization_error(response):
             attribute_value=nidcpower_types.NiDCPowerInt32AttributeValues.NIDCPOWER_INT32_MEASURE_WHEN_VAL_AUTOMATICALLY_AFTER_SOURCE_COMPLETE,
         )
     )
-    check_for_error(vi, configure_measure_when.status)
```

Comment:
I think we should keep these checks and helper, just change them to check_for_warning. If the status is > 0, we can get the error message and write to `sys.stderr`.

Reply:
Yep, I came to that conclusion eventually. And it's good because in a chat with Brad, he pointed out that the fetch operation can return a warning, so I added a call to it there too.